src/run/handlers/cache.cts

```diff
@@ -393,28 +401,28 @@ export class NetlifyCacheHandler implements CacheHandlerForMultipleVersions {
 
       await this.cacheStore.set(key, { lastModified, value }, 'blobStore.set')
 
-      if (data?.kind === 'PAGE' || data?.kind === 'PAGES') {
-        const requestContext = getRequestContext()
-        if (requestContext?.didPagesRouterOnDemandRevalidate) {
-          // encode here to deal with non ASCII characters in the key
-          const tag = `_N_T_${key === '/index' ? '/' : encodeURI(key)}`
-          const tags = tag.split(/,|%2c/gi).filter(Boolean)
-
-          if (tags.length === 0) {
-            return
-          }
+      // if (data?.kind === 'PAGE' || data?.kind === 'PAGES') {
```

Comment:
TODO - can we remove this if entirely, or do we need to make it less restrictive so 404 pages can purge cache as well?